.ruby-version

```diff
@@ -1 +1 @@
-3.2.0
+3.3.0
```

Comment:
Would be better to keep it at 3.2 to be consistent with Dockerfile

Reply:
Oh my mistake—I attempted to get to 3.3 and had to rollback. Forgot to do here—thanks!